src/App.js

```diff
@@ -1,231 +1,94 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
+import ProductDetail from './components/ProductDetail';
```

Comment:
Are you sure your app is working? When you import from a file, the file path is case sensitive.
What is the name of the folder or disk vs what you are importing?